Makefile

```diff
@@ -73,7 +75,7 @@ readinessServer: fmt vet
 	go build -o bin/readinessServer readinessProbe/main.go
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet manifests export_env_vars
+run: generate fmt vet manifests export_env_vars 
```

Comment:
install should be run here to install all the crds. Not sure if I should add with this PR or create a new one

Reply:
nvm, will send a different PR with other changes

Reply:
@ezio-auditore generally `install` will be run before `make run` manually.